src/main/java/hw_2/Task2.java

```diff
@@ -0,0 +1,14 @@
+package hw_2;
+
+public class Task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+
+        System.out.println(x + y);
+
+        System.out.println("" + x + y);
```

Comment:
There is should be the output in one line
![Screenshot_5](https://github.com/AntBelyi/ALevel_automation/assets/66726924/2044d896-5aab-4c6a-b4b4-b1722f1b3107)

Reply:
Done